### PR TITLE
Add netlist for power-ground aware GL simulation

### PR DESCRIPTION
**Pull request by ankita0805**
Add netlist for power-ground aware GL simulation.

### CODE REVIEW

**Comment by ctorng** [comment]
Ankita, does the .lvs.v not already have pg nets in it?

File: steps/cadence-innovus-signoff/scripts/generate-results.tcl [hunk 1]
```diff
@@ -53,6 +53,11 @@ saveNetlist -excludeLeafCell                        \
 
 saveNetlist -excludeLeafCell $vars(results_dir)/$vars(design).vcs.v
 
+# Write netlist for Power-Ground Aware GL simulation
+
+saveNetlist -includePowerGround $vars(results_dir)/$vars(design).pg.v
+
```
Comment (ctorng) [inline steps/cadence-innovus-signoff/scripts/generate-results.tcl:59]: Note from the call: Make it $vars(design).vcs.pg.v

Comment (ankita0805) [reply steps/cadence-innovus-signoff/scripts/generate-results.tcl:59]: Updated as discussed.

**Comment by ctorng** [comment]
I think you also need to add it as a new output in configure.yml.

**Comment by ankita0805** [comment]
okay, done. Can you check?

**Pull request merged**